### ISSUE TITLE
[Plugin request] Things3 integration

### ISSUE THREAD

**Issue by mortela-ai** [BasedHardware/omi#1016]
**Is your feature request related to a problem? Please describe.**
Create integration/plugin to Things3 app, https://culturedcode.com/things/ so that tasks identified in chat or memories, are created automatically to inbox.
Things provide url scheme that could be utilised to to create and update tasks https://culturedcode.com/things/support/articles/2803573/

**Describe the solution you'd like**
I would like to ask in chat what my tasks today are, and chat should be able to list any open tasks based on the date they are due. Also, when memories are handled, any task should be recorded to the Things inbox

**Describe alternatives you've considered**

**Additional context**

**Comment by j3sk0** [comment]
It's a pity that things is not available for android

**Comment by mortela-ai** [comment]
> It's a pity that things is not available for android

I agree with that. I think it has been one of the best task management apps I have tried, and easy to integrate with other apps
